wordpress-mobile/gutenberg-mobile
# Update GB reference.

Fixes #1929 and #1613 

GB PR: https://github.com/WordPress/gutenberg/pull/20288

To test:
 - Open the demo app
 - Add emoji to the title of the post for example: `🌈Testing!🌈`
 - Switch to HTML mode
 - Check that no extra `strong` element are added to the title.

PR submission checklist:
